tests/integration/key_assignment.go

```diff
@@ -142,6 +161,9 @@ func (s *CCVTestSuite) TestKeyAssignment() {
 				if err != nil {
 					return err
 				}
+
+				// check that the second key was also assigned correctly
```

Comment:
Forgot to remove a comment here :)

Reply:
Yes, my mistake. I removed the unnecessary comment.